addons/mail/static/src/core/common/thread_service.js

```diff
@@ -469,6 +469,7 @@ export class ThreadService {
     async joinChat(id) {
         const data = await this.orm.call("discuss.channel", "channel_get", [], {
             partners_to: [id],
+            force_open: true,
```

Comment:
Shouldn't we move up this variable to where the method is called?
As seen together, we wouldn't want to open the chat window from discuss app or other places, but limited to chat window new message.